numpyro/distributions/transforms.py

```diff
@@ -1021,6 +1120,13 @@ def forward_shape(self, shape):
     def inverse_shape(self, shape):
         raise NotImplementedError
 
+    def tree_flatten(self):
+        # XXX: what if unpack_fn is a parametrized callable pytree?
```

Comment:
I guess this is fine. I don't think we have usage case where unpack_fn depends on extra jax arrays.

Reply:
Agreed. Would you like me to downgrade this `#XXX`  to a simple `# note`, or to remove it entirely?

Reply:
I think just leave this as is.